python/tk_multi_workfiles/work_files.py

```diff
@@ -72,7 +73,7 @@ def __init__(self, app):
         self._work_area_template = None
         self._publish_template = None
         self._publish_area_template = None
-        
+
```

Comment:
yeah I agree ; I shouldn't have committed that. I had firstly done some other change here and there but my pycharm has an auto-del-space-from-empty-lines and anyway I shouldn't have commit that.
I'll fix.